turbopack/crates/turbopack-ecmascript/src/references/esm/base.rs

```diff
@@ -94,6 +94,10 @@ pub struct EsmAssetReference {
     pub origin: Vc<Box<dyn ResolveOrigin>>,
     pub request: Vc<Request>,
     pub annotations: ImportAnnotations,
+    /// True if the import should be ignored
+    /// This can happen for example when the webpackIgnore or turbopackIgnore
+    /// directives are present
+    pub ignore: bool,
```

Comment:
That looks unused, could we remove that?

Reply:
Good catch, I think this was left over from my refactoring efforts.